pkg/basicstation/cups/update_info.go

```diff
@@ -141,29 +143,29 @@ var getGatewayMask = pbtypes.FieldMask{Paths: []string{
 }}
 
 // UpdateInfo implements the CUPS update-info handler.
-func (s *Server) UpdateInfo(c echo.Context) (err error) {
-	// This is to account for older LBS gateways that don't set this header.
-	c.Request().Header.Set(echo.HeaderContentType, "application/json")
```

Comment:
We don't need this anymore?

Reply:
This was there for Echo's `Bind`, such that it knows that it should unmarshal the body using the JSON unmarshaler. We manually unmarshal the body now, so there is no need to pre-fill this anymore.